app/models/concerns/vacancy_phaseable.rb

```diff
@@ -0,0 +1,25 @@
+module VacancyPhaseable
```

Comment:
The name suggest you want to have it in namespace Vacancy::Phaseable. Or you can just call it EducationPhaseable